dq-server/src/controllers/fileImporter.js

```diff
@@ -0,0 +1,102 @@
+const { Question } = require('../db');
+const _ = require('lodash');
+const csv = require('csv-parser');
+const fs = require('fs');
+const { getThemes } = require('./themes');
+const { createQuestion, getQuestionByText } = require('./questions');
+const { createQuestionSchema } = require('../validation/input');
+
+async function importQuestions(stream) {
+	const streamData = await readCSVStream(stream, verifyCSVLine, () => {});
```

Comment:
As I said before. You will be able to do

readCSVStream(stream, (error) => {  return errors...}), (success) => { ... }))